_sass/components/_details.scss

```diff
@@ -0,0 +1,91 @@
+details.isomer-details summary {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  &:empty {
+    justify-content: flex-end;
+  }
+}
+
+details.isomer-details summary::marker {
+  display: none;
+}
+
+details.isomer-details summary::after {
+  // TODO: Think of a better way to do this
+  content: url("https://raw.githubusercontent.com/atisawd/boxicons/master/svg/solid/bxs-chevron-down.svg");
+  transition: 0.25s transform ease;
+  padding-top: 0.25rem;
+}
+
+details.isomer-details[open] summary::after {
+  transform: rotate(180deg);
+}
+
+@keyframes details-show {
+  from {
+    opacity: 0;
+    transform: var(--details-translate, translateY(-0.5em));
+  }
+}
+
+details.isomer-details[open] > *:not(summary) {
+  animation: details-show 150ms ease-in-out;
+}
+
+details.isomer-details {
+  display: flex;
+  border-top: 1px solid #d4d4d4;
+  border-bottom: 1px solid #d4d4d4;
+
+  > div {
+    flex: 1 1 auto;
+  }
+
+  div.isomer-details-content > p {
+    color: var(--site-secondary-color);
+    margin-left: 1rem;
+    margin-right: 1.25rem;
+    margin-bottom: 1rem;
+  }
+}
+
+details.isomer-details ~ details.isomer-details {
+  margin-top: 0;
+  margin-bottom: 0;
+  border-top: 0;
+}
+
+.isomer-accordion-group {
+  margin-bottom: 2rem;
+  margin-top: 2rem;
+}
+
+.isomer-accordion-group:first-child {
+  margin-top: 0;
+}
+
+.isomer-accordion-group:last-child {
+  margin-bottom: 0;
+}
+
+.isomer-details:last-of-type {
+  margin-bottom: 0;
+  border-bottom: 1px solid #d4d4d4;
+}
+
+.isomer-accordion-white > details.isomer-details > summary {
+  background-color: #fff;
+  color: #4372d6;
+}
+
+.isomer-accordion-gray > details.isomer-details > summary {
+  background-color: #f4f4f4;
```

Comment:
nit: can put these in colours

Reply:
hey @seaerchin ok to put in a separate pr? some tokens not defined in yet

Reply:
@sehyunidaaa - do we have the tokens?